packages/web-components/fast-foundation/src/select/select.pw.spec.ts

```diff
@@ -158,9 +157,9 @@ test.describe.only("Select", () => {
             `;
         });
 
-        await expect(element).toHaveJSProperty("value", "");
+        await expect(element).not.hasAttribute("value");
```

Comment:
This `hasAttribute` assertion is custom-made to work around a Playwright bug with empty attributes. It's been fixed in recent versions of Playwright, so I'll set this back to `toHaveAttribute` in microsoft/fast#6876.